akka-docs-dev/rst/scala/code/docs/stream/ReactiveStreamsDocSpec.scala

```diff
@@ -135,4 +133,16 @@ class ReactiveStreamsDocSpec extends AkkaSpec {
     assertResult(storage)
   }
 
+  "use a processor" in {
+
+    //#use-processor
+    // An example Processor factory
+    def createProcessor: Processor[Int, Int] = Flow[Int].toProcessor.run()
+
+    val src: Source[Int, Unit] = Source.single(42).viaProcessor(() => createProcessor)
+    val flow: Flow[Int, Int, Unit] = Flow[Int].viaProcessor(() => createProcessor)
```

Comment:
I disagree: *everything* on Flow is via. map, filter, the whole shebang.

@drewhk You mean `Source.via(Flow.wrap(processor))`? That would be OK if Source.wrap(publisher) and Sink.wrap(subscriber) would be the way to hoist those (symmetry and consistency ftw).

Flow.process(() => mkProcessor()) would be symmetric to Flow.transform(() => mkStage())

Or, perhaps what we have realized is that `transform` is the wrong name.